src/main/java/Greet.java

```diff
@@ -0,0 +1,140 @@
+        System.out.println("Here is the list of tasks");
+        for (int i = 0; i < listCounter; i++) {
+            System.out.println( i+1 + ". " + taskList.get(i).toString());
+        }
+        System.out.println("there are "+ (listCounter) +" tasks in the list");
+    }
+
+    private static void doTask(String[] command) {
+        int value;
+        try {
+            value=Integer.parseInt(command[1]);
+        } catch(IndexOutOfBoundsException e) {
+            System.out.println("index required");
+            return;
+        }
+        taskList.get(value-1).setDone(true);
+        System.out.println("Nice! I've marked this task as done: "+ taskList.get(value-1).toString());
+    }
+
+    public static void includeTask(String args){
+        String[] sections = args.split(" ");
+        int descriptionStart = args.indexOf(" ")+1;
+        String description=args;
+        switch (sections[0]){
+        case "deadline":{
+            int dividerPosition = args.indexOf("/");
+            try {
+                description= args.substring(descriptionStart,dividerPosition-1);
+            } catch(StringIndexOutOfBoundsException e){
+                System.out.print("____________________________________________________________\n" +
+                        "☹ OOPS!!! The description of a deadline or the time cannot be empty.\n" +
+                        " ____________________________________________________________\n");
+                return;
+            }
+            taskList.add(new Deadline(description,args.substring(dividerPosition+4)));
+            break;
+        }
+        case "todo":{
+            try {
+                description=sections[1];
+                description=args.substring(descriptionStart);
+            } catch(IndexOutOfBoundsException e){
+                System.out.print("____________________________________________________________\n" +
+                        "☹ OOPS!!! The description of a todo cannot be empty.\n" +
+                        " ____________________________________________________________\n");
+                return;
+            }
+
+            taskList.add(new Todo(description));
+            break;
+        }
+        case "event":{
+            int dividerPosition = args.indexOf("/");
+            try {
+                description= args.substring(descriptionStart,dividerPosition-1);
+            } catch(StringIndexOutOfBoundsException e){
+                System.out.print("____________________________________________________________\n" +
+                        "☹ OOPS!!! The description of an event or the time cannot be empty.\n" +
+                        " ____________________________________________________________\n");
+                return;
+            }
+
+            taskList.add(new Event(description,args.substring(dividerPosition+4)));
+            break;
+        }
+        default: {
+            //taskList[listCounter] = new Task(args);
```

Comment:
Avoid leaving commented code in the repo.

If you ever need this code fragment, you can retrieve it using revisoin control.

The issue with leaving commented code in the repo is that it may permanently be left in there.